ui/analyse/src/main.ts

```diff
@@ -22,7 +21,3 @@ function userAnalysis(cfg: any) {
   cfg.socketSend = lichess.socket.send;
   const analyse = start(cfg);
 }
-
-// that's for the rest of lichess to access chessground
-// without having to include it a second time
-window.Chessground = Chessground;
```

Comment:
Please don't remove this! :) I use it in LiChess Tools.

Reply:
I'm not crazy about unnecessary window globals in lila. Is it just in analysis where you need it?

Reply:
I agree with you, but you could put it in the lichess object. Right now I am using it to draw arrows for mobile in TV games and during play, since the chessground instance is not accessible I am creating a transparent new chessground to draw arrows on.

It is a kind of a hack, but the only solution I could find sans rewriting the arrow drawing code from scratch.